File: Java/Assignment_4/src/com/vti/entity/Position.java
package com.vti.entity;

public class Position {
	short positionID;
	enum_position positionName;
	
	
	public short getPositionID() {
		return positionID;
	}
	public void setPositionID(short positionID) {
		this.positionID = positionID;
	}
	public enum_position getPositionName() {
		return positionName;
	}
	public void setPositionName(enum_position positionName) {
		this.positionName = positionName;
	}
	
}

File: ReactJS/Assignment_1/Exercise_1/Position.ts
import { Posi } from "./enumPosition";

export class Position{
    private static counter : number = 0;
    PositionID : number ;
    PositionName : Posi;


	constructor(PositionName : Posi) {
        this.PositionID = ++Position.counter;
        this.PositionName = PositionName;
    }

    public setPositionName() : Posi{
        return this.PositionName;
    }
    
    public getPositionName(PositionName : Posi){
        this.PositionName = PositionName;
    }

    public printPosition(){
        console.log("ID position là: " + this.PositionID);
        console.log("Name position là: " + this.PositionName);
    }

}